### PR TITLE
phpstan: fixed "Access to an undefined property"

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Set undefined properties. I hope its correct to make "new" properties public. 

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

**Comment by fballiano** [comment]
why not protected?

**Comment by elidrissidev** [comment]
They should probably be protected by default unless there is a usage outside the class.

**Comment by sreichel** [comment]
> They should probably be protected by default unless there is a usage outside the class.

Yes, BUT ... they were public all the time and someone could have used them outside the class.

I can change it, because they should have not been used outside. Let me know.

**Comment by fballiano** [comment]
right, but php 8.2 deprecates public properties so...
https://php.watch/versions/8.2/dynamic-properties-deprecated
what's the right choice here?

**Comment by sreichel** [comment]
Public is still okay, but not dynamically set.

File: app/code/core/Mage/Adminhtml/Block/Widget/Grid/Container.php [hunk 1]
```diff
@@ -31,6 +31,14 @@ class Mage_Adminhtml_Block_Widget_Grid_Container extends Mage_Adminhtml_Block_Wi
     protected $_backButtonLabel;
     protected $_blockGroup = 'adminhtml';
 
+    /**
+     * @var string
+     */
+    public $_block;
```
Comment (fballiano) [inline app/code/core/Mage/Adminhtml/Block/Widget/Grid/Container.php:37]: could this be protected?

File: app/code/core/Mage/Catalog/Model/Resource/Category/Tree.php [hunk 2]
```diff
@@ -67,9 +67,13 @@ class Mage_Catalog_Model_Resource_Category_Tree extends Varien_Data_Tree_Dbp
      */
     protected $_storeId                          = null;
 
+    /**
+     * @var array
+     */
+    public $_inactiveItems;
```
Comment (fballiano) [inline app/code/core/Mage/Catalog/Model/Resource/Category/Tree.php:73]: and this?

Comment (sreichel) [reply app/code/core/Mage/Catalog/Model/Resource/Category/Tree.php:73]: Missed both. Thanks.

**Review by fballiano — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Pull request merged**